mockwebserver/src/main/kotlin/mockwebserver3/SocketPolicy.kt

```diff
@@ -51,12 +51,6 @@ enum class SocketPolicy {
    */
   DISCONNECT_AT_END,
 
-  /**
-   * Wrap the socket with SSL at the completion of this request/response pair. Used for CONNECT
-   * messages to tunnel SSL over an HTTP proxy.
-   */
-  UPGRADE_TO_SSL_AT_END,
```

Comment:
This is an OkHttp 5 backwards-incompatible change!

Reply:
Seems reasonable, it's not a dependency of libraries, more likely pulled into app in tests.

5 is the time to do it.